File: GameOfDrones.Models/Round.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace GameOfDrones.Models
{
    public class Round 
    {
        public int RoundId { get; set; }

        public int? WinnerPlayerId { get; set; }
        public Player Winner { get; set; }

        [JsonIgnore]
        public Game Game { get; set; }
        public int GameId { get; set; }

        public ICollection<PlayerMove> Moves { get; set; }

    }
}

File: GameOfDrones.UI/src/models/Round.ts
import { Player } from './Player';
import { PlayerMove } from './PlayerMove';

export class Round {
	roundId: number;
	number: number;
	gameId: string;
	moves: Array<PlayerMove>;
	winner: Player;

	constructor(gameId: string, number: number) {
		this.gameId = gameId;
		this.number = number;
		this.moves = new Array<PlayerMove>();
	}
}